tree/30_generic_methods/sharedfile_to_node.cf

```diff
@@ -118,6 +118,11 @@ bundle agent sharedfile_to_node(target_uuid, file_id, file_path, ttl)
 
       "url_ttl"           string => execresult("echo ${ttl} | sed 's/ /+/g'", "useshell");
 
+    inventory_proxy_defined::
+      "curl_proxy"        string => "";
+    !inventory_proxy_defined::
+      "curl_proxy"        string => "${node.properties[inventory_proxy]}";
```

Comment:
* Why inventory_proxy?
* This really needs to be documented, in a section about "special variables and parameters"

Reply:
We will do https://www.rudder-project.org/redmine/issues/12100 instead